wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -941,15 +933,12 @@ public void onTrimMemory(final int level) {
         }
     }
 
-    private Notification createNetworkSyncNotification(BlockchainState blockchainState) {
+    private Notification createNetworkSyncNotification() {
```

Comment:
We should keep the blockchainState parameter for the reasons mentioned in the other comments.